app/controllers/code_schools_controller.rb

```diff
@@ -7,4 +7,16 @@ def index
     @code_schools_by_state = code_schools.by_state
     @country = Country.named('United States')
   end
+
+  def map
+    code_schools = CodeSchools.new.all
+    @lat_longs = []
+
+    code_schools.each do |hash| 
```

Comment:
Would it make sense to move this to the `CodeSchools` class? Maybe something like `CodeSchools.lat_longs`? I think it would make things easier to test and it would clean up the controller a bit.